src/main/java/edu/ie3/datamodel/models/value/GroundTemperatureValue.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * © 2025. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.models.value;
+
+import edu.ie3.datamodel.models.StandardUnits;
+import java.util.Objects;
+import java.util.Optional;
+import javax.measure.quantity.Temperature;
+import tech.units.indriya.ComparableQuantity;
+
+/** Describes a ground temperature value. */
+public class GroundTemperatureValue implements Value {
```

Comment:
I think you can reuse many parts from `TemperatureValue` if you make `GroundTemperatureValue` extending from it.